README.md

```diff
@@ -39,6 +39,10 @@ Installers
 Refer to the [downloads page][etcher] for the latest pre-made
 installers for all supported operating systems.
 
+#### AppImage
+
+Set execution permissions on the `.AppImage` file and double-click it.
```

Comment:
Should this say 'executable permissions' rather than 'execution permissions' ?

Reply:
See https://github.com/resin-io/etcher/pull/1715